metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFactory.java

```diff
@@ -95,7 +99,11 @@ private void init() {
     private IMetacatHiveClient createLocalClient() throws MetaException {
         final HiveConf conf = getDefaultConf();
         configuration.forEach(conf::set);
-        return new EmbeddedHiveClient(MetacatHMSHandler.newRetryingHMSHandler("metacat", conf, true));
+        DataSourceManager.get().load(catalogName, configuration);
```

Comment:
We will have to rethink the use of DataSourceManager. For now, lets add a TODO item.